src/drivers/actuators/vertiq_io/module.yaml

```diff
@@ -0,0 +1,271 @@
+module_name: Vertiq IO
+serial_config:
+    - command: vertiq_io telemetry ${SERIAL_DEV}
+      port_config_param:
+        name: VERTIQ_IO_CFG
+        group: Vertiq IO
+
+actuator_output:
+  output_groups:
+    - param_prefix: VERTIQ_IO
+      group_label: 'ESCs'
+      channel_label: 'ESC'
+      standard_params:
+        min: { min: 0, max: 65535, default: 0 }
+        max: { min: 0, max: 65535, default: 65535 }
+        failsafe: { min: 0, max: 500 }
+      num_channels: 16
+
+parameters:
+    - group: Vertiq IO
+      definitions:
+        VERTIQ_BAUD:
+            description:
+                short: The IQUART driver's baud rate
+                long: |
+                    The baud rate (in bits per second) used by the serial port connected with IQUART communication
+            type: int32
+            default: 115200
+            reboot_required: true
+        VERTIQ_NUM_CVS:
+            description:
+              short: The number of Vertiq IFCI parameters to use
+              long: |
+                  The total number of IFCI control variables being used across all connected modules
+            type: int32
+            default: 0
+            min: 0
+            max: 16
+            reboot_required: true
+        DISARM_BEHAVE:
+            description:
+                short: The triggered behavior on PX4 disarm
+                long: |
+                    The behavior triggered when the flight controller disarms. You have the option to trigger your motors' disarm behaviors, set all motors to coast,
+                    or set a predefined throttle setpoint
+            type: enum
+            values:
+                0: Trigger Motor Disarm Behavior
```

Comment:
Yes, right now, it just sends an explicit disarm. If users want to set up in order to use the predefined setpoint to disarm they could do that as well. Sending the explicit disarm is certainly easier to do out-of-box though.

Reply:
Got it. I like having this option around, I think it's name may just be a bit confusing, the phrase "Disarm Behavior" is a bit overloaded now, since we already use it on the RTD. If I were a customer reading this, my first thought my be that I had to use this option to trigger disarming behavior, which isn't necessarily true if you've set up your throttle stuff instead.

Might be better to change the name here to something like "Send Explicit Disarm Command" and to avoid the use of "Disarm Behavior".